models/DataObject/ClassDefinition/Data/Relations/AbstractRelations.php

```diff
@@ -73,6 +82,23 @@ public function setClasses($classes)
         return $this;
     }
 
+    /**
+     * @return string|null
+     */
+    public function getSqlCondition(): ?string {
+        return $this->sqlCondition;
+    }
+
+    /**
+     * @param string|null $sqlCondition
+     * @return AbstractRelations
+     */
+    public function setSqlCondition(?string $sqlCondition): AbstractRelations {
```

Comment:
I think this would be better:

```suggestion
     * @return $this
     */
    public function setSqlCondition(?string $sqlCondition): static {
```